docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
+``` xml
+IsPaneOpen="{Binding IsSplitViewPaneOpen}"
+```
+You should now have the following configuration:
+``` xml
+<SplitView PanePlacement="Right" DisplayMode="CompactInline" IsPaneOpen="{Binding IsSplitViewPaneOpen}">
+```
+
+## Configuring the button
+At this point all we need to do is make the button open and close the SplitView's pane when clicked.
+To do so create a public property of type ICommand in the ViewModel with just the "getter method".
+``` C#
+public ICommand ChangeSplitViewPaneStatusCommand { get; }
+```
+And initialize it in the constructor.
+``` C#
+this.ChangeSplitViewPaneStatusCommand = ReactiveCommand.Create(() =>
+{
+    this.IsSplitViewPaneOpen = !this.IsSplitViewPaneOpen;
+});
+```
+In this example I will be using a very simple logic: if the value is true it becomes false, and if it's false it becomes true.
+<br>
+You should now have the following code in the ViewModel:
+
+``` C#
+public class MainWindowViewModel : ViewModelBase
+{
+    private bool _isSplitViewPaneOpen;
+    public bool IsSplitViewPaneOpen
+    {
+        get => this._isSplitViewPaneOpen;
+        set
+        {
+            this.RaiseAndSetIfChanged(ref this._isSplitViewPaneOpen, value);
+        }
+    }
 
-TO DO
+    public ICommand ChangeSplitViewPaneStatusCommand { get; }
 
-<img src={SplitViewExpanderScreenshot} alt=""/>
+    public MainWindowViewModel()
+    {
+        this._isSplitViewPaneOpen = false;
+        this.ChangeSplitViewPaneStatusCommand = ReactiveCommand.Create(() =>
+        {
+            this.IsSplitViewPaneOpen = !this.IsSplitViewPaneOpen;
+        });
+    }
+}
+```
+Now, to make the button invoke the command you just need to bind it. (For more informations see [here](https://docs.avaloniaui.net/docs/0.10.x/data-binding/binding-to-commands))
+``` xml
+<Button Command="{Binding ChangeSplitViewPaneStatusCommand}">
+```
+By doing so the command will be invoked each time the button is clicked and will update the IsSplitViewPaneOpen's value accordingly.
+<br><br>
+**If you followed this guide step by step you should now have something like this:**
+<img className="screenshot-full" src={SecondExample} alt="Second example" />
 
+## Adding content to the button
+Finally, you now just need to add some content to the button that indicates whether you need to show or hide the SplitView's pane. In this guide we will be using the following characters:
+- '<' to indicate the pane will be shown on click.
+- '>' to indicate the pane will be hidden on click.
+
+### Implement a converter
+To do this step we will need the char to change as the Pane goes from shown to hidden and viceversa. To do so we need to create a converter function that will return the necessary char to the button when clicked.<br>
+In the `models` folder create a class named SplitViewIconConverter and make it inherit from the IMultiValueConverter interface.<br>
```

Comment:
Is the folder name actually lower case? I know this is somewhat pedantic but it does actually make a different on things like Linux with case sensitive file names